### PR TITLE
Add three functions to `presentation` namespace

### PR DESCRIPTION
**Pull request by james-d-mitchell**

### CODE REVIEW

**Review by MTWhyte — APPROVED** [review]
This all looks good to me!

**Comment by james-d-mitchell** [comment]
Except the CI failing :(

File: include/libsemigroups/present.hpp [hunk 1]
```diff
@@ -27,6 +27,7 @@
 
 #include <algorithm>         // for reverse, sort
 #include <cstddef>           // for size_t
+#include <cstring>           // for strlen
```
Comment (james-d-mitchell) [inline include/libsemigroups/present.hpp:30]:
```suggestion
#include <cstring>           // for strlen 
```

Comment (james-d-mitchell) [inline include/libsemigroups/present.hpp:30]:
```suggestion
#include <cstring>           // for strlen 
```

**Comment by james-d-mitchell** [comment]
I think the CI will fail until the next version of `libsemigroups_pybind11` is released, there's a PR open for that:

https://github.com/libsemigroups/libsemigroups_pybind11/pull/101

**Comment by james-d-mitchell** [comment]
So the CI for `libsemigroups_pybind11` is still broken, but this time for a good reason, this PR changes the test output. So, I'll have to make some further adaptations to `libsemigroups_pybind11` before this can be merged.

**Comment by james-d-mitchell** [comment]
Finally this can be merged :)

**Pull request merged**